src/web/src/Pages/NewChatPage/Components/Pay/PayModalFooter.tsx

```diff
@@ -2,28 +2,51 @@ import { memo } from 'react';
 import styles from './PayModalFooter.module.scss';
 import classNames from 'classnames';
 import { useTranslation } from 'react-i18next';
+
 export const PayModalFooter = ({ className }) => {
   const { t } = useTranslation();
-  return (
-    <div className={classNames(styles.protocolWrapper, className)}>
-      <div className={styles.protocolDesc}>{t('pay.protocolDesc')}</div>
-      <div className={styles.protocolLinks}>
+
+  const renderVirtualProductPoint1 = () => {
+    const point1Text = t('pay.virtualProductPoint1');
+    const parts = point1Text.split(/\{\{modelServiceAgreement\}\}|\{\{userPrivacyPolicy\}\}/g);
+
+    if (parts.length !== 3) {
+      return point1Text;
+    }
+
+    return (
+      <>
+        {parts[0]}
         <a
           className={styles.protocolLink}
           href="/useragreement"
           target="_blank"
           referrerPolicy="no-referrer"
+          rel="noopener"
         >
           {t('pay.modelServiceAgreement')}
         </a>
+        {parts[1]}
         <a
```

Comment:
To remove some whitespaces.

```suggestion
        </a>{parts[1]}<a
```